test/Test/Data/List.hs

```diff
@@ -41,3 +44,19 @@ take_length = property $ do
     False -> do
       annotate "Prelude.length xs < n"
       Prelude.length (List.take n xs) === Prelude.length xs
+
+zipWith_lazy :: Property
+zipWith_lazy = property $ do
+  _ <- eval $ Prelude.head xs
+  Prelude.return ()
+  where
+    xs :: [Word]
+    xs = List.zipWith (Num.+) (0 : error "bottom") [0 .. 42]
```

Comment:
What is the motivation for using static bounds for ranges instead of property-supplied ones?
(same below)